Library/Formula/chapel.rb

```diff
@@ -22,12 +19,15 @@ def install
     # https://gist.github.com/DomT4/90dbcabcc15e5d4f786d
     # https://github.com/Homebrew/homebrew/pull/35166
     cd libexec do
-      system "make", "all"
+      system "make"
+      system "make", "chpldoc"
     end
 
     prefix.install_metafiles
 
-    (bin/"chpl").write_env_script libexec/"bin/darwin/chpl", :CHPL_HOME => libexec
+    # Install chpl and other binaries (e.g. chpldoc) into bin/ as exec scripts.
```

Comment:
It's three programs: chpl, chpldoc, chpl-ipe.

chpldoc in the 1.10 release did not work in homebrew due to a few issues. We fixed those issues and added a bunch of features during this last cycle, so I think it makes sense to include it. chpl-ipe is the new interpreter for chapel, and I'd like to make it available to users.

I'm happy to call them out individually, if that is preferred.

Thanks!